crates/lib/Cargo.toml

```diff
@@ -14,6 +14,7 @@ manual-tests = []
 tracing = ["dep:tracing", "qcs-api-client-common/tracing", "qcs-api-client-grpc/tracing", "qcs-api-client-openapi/tracing"]
 tracing-config = ["tracing", "qcs-api-client-common/tracing-config", "qcs-api-client-grpc/tracing-config", "qcs-api-client-openapi/tracing-config"]
 otel-tracing = ["tracing-config", "qcs-api-client-grpc/otel-tracing", "qcs-api-client-openapi/otel-tracing"]
+libquil = ["libquil-sys"]
```

Comment:
Should we make this a default feature?

Reply:
Oh, interesting. I didn't realise there were implicit features. Thanks.